Dropdown (#6310)
SkylineCommunications/dataminer-docs: dataminer/Functions/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/Dropdown.md

# Dropdown

This basic control allows you to **select an item in a dropdown list**. The selectable items can be based on any data.

This component is typically used to let users interact with a dashboard or low-code app by selecting a value that influences what is shown elsewhere. For example, a dropdown can be used to filter data in other components, switch between different data sets, or focus on a specific element, protocol, or status. This makes it easier to explore large amounts of data and quickly find the information that is most relevant.

![Dropdown](~/dataminer/images/Dropdown.png)<br>*Dropdown component in DataMiner 10.4.6*

> [!NOTE]
> Prior to DataMiner 10.3.5/10.4.0<!--  RN 35902 -->, this component is available under *Feeds*.

> ## Data source behavior

Reviewer: Not sure if we'd already use "Data source behavior" anywhere in our docs. To me this sounds like the behavior of the data source although I have no better alternative.

Author: Maybe this is better?
```suggestion
## How data is displayed in the dropdown
```

1. Click *Select operator* and select *Filter*.

1. Select the column you want to filter.

1. Set the filter method to *Equals* or *Contains*.

1. On the right side of the *Value* box, click the ![Link to](~/dataminer/images/Link_to_Data.png) icon to open the *Link to* dialog.

1. Configure the following settings:

   - *Data*: Select the dropdown component.

   - *Property*: Select the column that contains the value to filter on.

   - *Type*: `Tables`.

   - *Empty data shows*: Define what happens when no value is selected in the dropdown:

     - *Everything*: Display all data.

     - *Nothing*: Display no data.

1. Click *Apply*.

1. Click the pencil icon again to stop editing the query.

> [!NOTE]
> The *Tables* data source (used here to link dropdown values to other components) is available from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41161-->.

## Configuration options

### Dropdown layout

In the *Layout* pane, you can find the default options available for all components. See [Customizing the component layout](xref:Customize_Component_Layout).

Additionally, the following layout options are also available:

| Section | Option | Description |
|--|--|--|

> | Advanced | Display column | Available from DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->. Select which column is used to display as the display value in the dropdown. Only available when a single query data source or *Tables* component data source was applied. |

Reviewer: "Select which column is used to display as the display value in the dropdown." does not read that nicely.

Author: Is this better?
```suggestion
| Advanced | Display column | Available from DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->. Select the column whose values are shown in the dropdown. Only available when a single query data source or *Tables* component data source was applied. |
```
